image/templates/helm/stackrox-secured-cluster/internal/defaults/50-images.yaml.htpl

```diff
@@ -60,13 +57,12 @@ image:
     {{- else }}
     repository: {{ list ._rox.image.collector.registry ._rox.image.collector.name | compact | join "/" }}
     {{- end }}
+  # TODO: Shouldn't [<- if not .KubectlOutput >] be here?
```

Comment:
?
Before merge please add a ROX issue

Reply:
The hope is for someone from Draco to see this and answer this in this PR, so I can remove the TODO prior to merge

Reply:
I agree, this seems odd.
It looks like it was introduced in commit https://github.com/stackrox/stackrox/commit/bb4100cf22980001b6fb62906049a8baafbd1375. But I don't understand yet why.